### PR TITLE
Add fetching podCIDR and serviceCIDR from nsmd-k8

### PR DESCRIPTION
**Pull request by lobkovilya**
Signed-off-by: Ilya Lobkov <lobkovilya@yandex.ru>

## Description

Add service:
```
service ClusterInfo {
    rpc GetClusterConfiguration(google.protobuf.Empty) returns (ClusterConfiguration);
}
```
It allows to fetch podSubnet(podCIDR) and serviceSubnet(serviceCIDR) of kubernetes cluster.

## Motivation and Context

Covers https://github.com/networkservicemesh/networkservicemesh/issues/839 and https://github.com/networkservicemesh/networkservicemesh/issues/840
## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [X] Added integration testing to cover
- [X] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: k8s/conf/cluster-role-admin.yaml [hunk 1]
```diff
@@ -16,3 +16,6 @@ rules:
   - apiGroups: ["apiextensions.k8s.io"]
     resources: ["customresourcedefinitions"]
     verbs: ["*"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get"]
```
Comment (nickolaev) [inline k8s/conf/cluster-role-admin.yaml]: Interesting, why is that needed?

Comment (lobkovilya) [reply k8s/conf/cluster-role-admin.yaml]: It's needed to give nsmd-k8s ability to "get" configmap `kubeadm-config` from `kube-system` namespace. 
I think later we need more accurate role management and its binding.

File: test/integration/basic_nsmd_k8s_test.go [hunk 2]
```diff
@@ -302,3 +302,48 @@ func getNsmUrl(discovery registry.NetworkServiceDiscoveryClient) string {
...
+	if testing.Short() {
+		t.Skip("Skip, please run without -short")
+		return
+	}
+
+	k8s, err := kube_testing.NewK8s()
+	defer k8s.Cleanup()
+	Expect(err).To(BeNil())
+
+	k8s.Prepare("nsmd")
```
Comment (nickolaev) [inline test/integration/basic_nsmd_k8s_test.go]: Please merge latest master.We call the pod `nsmgr` now.

File: test/integration/basic_nsmdp_test.go [hunk 3]
```diff
@@ -23,15 +24,15 @@ func TestNSMDDP(t *testing.T) {
 
 	Expect(err).To(BeNil())
 
-	k8s.Prepare("nsmd", "icmp")
+	k8s.Prepare("dataplane", "icmp", "nsmgr")
```
Comment (nickolaev) [inline test/integration/basic_nsmdp_test.go]: Apparently, we have a couple of these left all over. I'll fix these in a separate PR.
